src/app/legacy/components/Promo/image.jsx

```diff
@@ -20,38 +20,61 @@ const ChildWrapper = styled.div`
 
 const WEBP_ORIGIN_CODES = ['cpsdevpb', 'cpsprodpb'];
 
-const createSrcSet = (imageUrl, suffix = '') => {
+// promos with images via Programmes (which can be of type audio and possibly others) use a different iChef recipe requiring a second set of resolutions
+// https://github.com/bbc/programme-images/tree/master/webapp/ichef/recipes
+const createSrcSet = (imageUrl, suffix = '', isProgrammeImage) => {
   const imageResolutions = [85, 120, 170, 232, 325, 450, 660];
+  const imageResolutionsProgrammes = [96, 128, 176, 240, 352, 464, 672];
 
-  return imageResolutions
+  const resolutions = isProgrammeImage
+    ? imageResolutionsProgrammes
+    : imageResolutions;
+
+  return resolutions
     .map(res => `${imageUrl.replace(`{width}`, res)}${suffix} ${res}w`)
     .join(', ');
 };
 
-const createSizes = useLargeImages => {
+const createSizes = (useLargeImages, isProgrammeImage) => {
   // 4 columns of fixed width
   const DESKTOP_SIZE = useLargeImages
     ? `(min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) 660px`
     : `(min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) 232px`;
 
+  const DESKTOP_SIZE_PROGRAMMES = useLargeImages
+    ? `(min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) 672px`
+    : `(min-width: ${GEL_GROUP_4_SCREEN_WIDTH_MIN}) 240px`;
+
   // 2 columns of 50% screen width - images are 100% of the column
   const TABLET_SIZE = `(min-width: ${GEL_GROUP_3_SCREEN_WIDTH_MIN}) 50vw`;
 
   // 1 column of 100% screen width - images are 33% of the column
   const MOBILE_SIZE = `33vw`;
-  return [DESKTOP_SIZE, TABLET_SIZE, MOBILE_SIZE].join(', ');
+
+  return [
+    isProgrammeImage ? DESKTOP_SIZE_PROGRAMMES : DESKTOP_SIZE,
+    TABLET_SIZE,
+    MOBILE_SIZE,
+  ].join(', ');
 };
 
 const Image = props => {
   const { children, src, useLargeImages, ...rest } = props;
   const isWebPSupported = WEBP_ORIGIN_CODES.some(originCode =>
     src.includes(originCode),
   );
-  const primarySrcSet = createSrcSet(src, isWebPSupported ? '.webp' : '');
+  const isProgrammeImage = src.includes('https://ichef.bbci.co.uk/images/ic/');
```

Comment:
Would `src.startsWith` also cause an alert?
```suggestion
  const isProgrammeImage = src.startsWith('https://ichef.bbci.co.uk/images/ic/');
```

Reply:
Might actually fix it, i'll commit and see if the browser behaviour is the same 👍

Reply:
Just checked it and this should be good so will see if the GH actions all pass 👍